frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -0,0 +1,56 @@
+# Contributing Data
+
+Cellxgene has a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you feel like you have a single cell dataset that meets our requirments (see below), then feel free to reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com).
+
+## Publishing Process
+
+The general process for submission to the portal looks like this:
+- reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) with a description of the dataset that you'd like to contribute. Generally speaking this would be derived from either human or mouse. We not allow submission of data sets from:
+  - organisms other than human or mouse
+  - cell lines
+  - drug screens 
+- recieve confirmation form the curation team will accept your dataset along with instructions on how to format your dataset for submission.
+- Prepare your dataset and email our curators for further assistance in the process
+- Publish!
```

Comment:
This section conflates data eligibility and data publishing, which has created confusion. We should put eligibility criteria into "requirements", as indicated in the first paragraph, and only talk about the submission process here. 

I've addressed the formatting/spelling (!) issues above with my suggestion. 

I discuss the eligibility criteria in a separate comment, below. 

```suggestion
## Publishing Process

The process for submission to the portal is:
- You confirm we can support your submission by reaching out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) with a description of the data that you'd like to contribute.
- We confirm that we will accept your data.
- You prepare your data according to our submission [requirements](#dataset-requirements) and send us your files.
- We upload to a private collection where you can review.
- We publish the data when you tell us to.
```

@jahilton does this text reflect how you'd like to see the process described?

Reply:
@jahilton